osf/models/node.py

```diff
@@ -656,10 +656,14 @@ def csl(self):  # formats node information into CSL format for citation parsing
             csl['DOI'] = doi
 
         if self.logs.exists():
-            citation_date = self.logs.latest().date
+            citation_date = self.date_modified
```

Comment:
Could you really quick check with @mfraezz and see if there's a way to do this that will work better with his upcoming change? There might be an alternate field in the near future that you could use instead.

Reply:
Currently, the future-proof way is to keep it as `logs.latest().date`. 

In the hopefully near future, there will be a `last_logged` field that prevents having to do a related lookup, but this will break when `date_modified` becomes `modified`. Then again, so will `preprint.date_modified` below. Please ensure that conditional branch is covered by a test